pkg/charts/linkerd2/values_test.go

```diff
@@ -158,10 +159,11 @@ func TestNewValues(t *testing.T) {
 	t.Run("HA", func(t *testing.T) {
 		actual, err := NewValues(true)
 
-		// workaround for mergo, which resets these to []interface{}(nil)
-		// and []PrometheusRuleConfigMapMount(nil)
+		// workaround for mergo, which resets these to []interface{}(nil),
+		// []PrometheusRuleConfigMapMount(nil) and Persistence(nil)
 		actual.PrometheusAlertmanagers = []interface{}{}
 		actual.PrometheusRuleConfigMapMounts = []PrometheusRuleConfigMapMount{}
+		actual.PrometheusPersistence = Persistence{}
```

Comment:
I don't really understand the original comment or the mergo issue, but are you sure that it applies to Persistence?  I notice that the other two fields are slices and this one is not.

Reply:
Nor do I (@alpeb could comment) and no I am not sure that it does, I was just trying stuff to get tests to pass 🤷‍♂️ 🙈 I will remove this, I think you pointed out the reason for failing tests in another comment.